src/Symfony/Component/Security/Http/RememberMe/TokenBasedRememberMeServices.php

```diff
@@ -98,7 +98,7 @@ protected function onLoginSuccess(Request $request, Response $response, TokenInt
      *
      * @return string
      */
-    protected function generateCookieValue($class, $username, $expires, $password)
+    protected function generateCookieValue(string $class, string $username, int $expires, string $password)
```

Comment:
Some phpdoccs can be removed. As a rule of thumb, any phpdoc that does not add any information not contained in the method signature can be removed.